app/v3/controllers/RetrieveSelfEmploymentBsasController.scala

```diff
@@ -51,14 +51,14 @@ class RetrieveSelfEmploymentBsasController @Inject()(
       endpointName = "retrieve"
     )
 
-  def retrieve(nino: String, calculationId: String): Action[AnyContent] =
+  def retrieve(nino: String, calculationId: String, taxYear: Option[String] = None): Action[AnyContent] =
```

Comment:
I think it's better to rename `retrieve` method to `handleRequest` just so it is consistent with the other endpoints